sentry-spring-boot-starter/src/test/kotlin/io/sentry/spring/boot/SentryLogbackAppenderAutoConfigurationTest.kt

```diff
@@ -29,16 +29,24 @@ class SentryLogbackAppenderAutoConfigurationTest {
     }
 
     @Test
-    fun `configures SentryAppender`() {
+    fun `hub is not created when auto-configuration dsn is not set`() {
         contextRunner
+            .run {
+                assertThat(it).doesNotHaveBean(IHub::class.java)
```

Comment:
Wouldn't it make more sense to verify if the appender is configured?

```java
assertThat(rootLogger.getAppenders(SentryAppender::class.java)).isEmpty()
```

Reply:
Yep, I'm sure it does :-) Done.

Reply:
Fantastic, thank you!